projects/igniteui-angular/src/lib/test-utils/grid-functions.spec.ts

```diff
@@ -1780,6 +1781,78 @@ export class GridFunctions {
         return headerTitle.parent;
     }
 
+    public static verifyGridPager(fix, rowsCount, firstCellValue, pagerText, buttonsVisibility) {
```

Comment:
This method should be in the file grid.pagination.spec.ts. It is used only there and for a specific grid definition

Reply:
Done